Client/iTeamTalk/iTeamTalk/ChannelListViewController.swift

```diff
@@ -883,6 +884,17 @@ class ChannelListViewController :
                 self.tableView.reloadData()
             }
             
+        case CLIENTEVENT_CMD_MYSELF_KICKED :
+            playSound(.srv_LOST)
+            tableView.reloadData()
+            if #available(iOS 8.0, *) {
+                let alert = UIAlertController(title: NSLocalizedString("Error", comment: "Dialog"), message: NSLocalizedString("You have been kicked from channel", preferredStyle: UIAlertController.Style.alert)
```

Comment:
You should check "nSource" to see if you were kicked from channel or server.
https://github.com/BearWare/TeamTalk5/blob/fa14b4c1fa46386f5d9ef5fa774acc637a1ba9f7/Library/TeamTalk_DLL/TeamTalk.h#L3137